packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1866,6 +1866,8 @@ export class AccountHandler {
       reason: ReasonForDeletion.UserRequested,
     });
 
+    this.glean.account.deleteComplete(request, { uid: accountRecord.uid });
```

Comment:
@vpomerleau This metric won't get emitted when an account is deleted from by admin or cloud task. I think that is fine since the intended behavior is to track the user flow through settings?

Reply:
Oh, that's a good question. I'll double check with Ross to confirm this is what we want.

Reply:
Ross confirmed that this event should only fire when it’s initiated by the user, so we are good to go! Thanks for raising the question.